README.md

```diff
@@ -1,43 +1,73 @@
-[Docker](https://docker.io) image bundling all the components and dependencies
-of the [Google Cloud SDK](https://cloud.google.com/sdk/) including alpha and
-beta components.
+
+This is Docker image for the [Google Cloud SDK](https://cloud.google.com/sdk/).
+
+Image includes default command line tools:
+*  gcloud:  Default set of gcloud commands
+*  gsutil:  Cloud Storage Command Line Tool
+*  bq: BigQuery Command Line Tool
+
+## Supported tags and respective Dockerfile links
+
+* 155.0.0, latest (Dockerfile)
 
 ## Usage
 
-Follow these instructions if you are running docker *outside* of Google Compute Engine:
+To use the prebuilt **Alpine** based image, pull and run from
+
+* [https://hub.docker.com/r/google/cloud-sdk/](https://hub.docker.com/r/google/cloud-sdk/)
+
+The docker images do not have an ENTRYPOINT so you can call gcloud directly as shown below:
+
+```bash
+docker run -ti  google/cloud-sdk:latest gcloud version --format='value("Google Cloud SDK")'
+155.0.0
+```
+
+or via tagged version of the SDK:
+
+```bash
+docker run -ti google/cloud-sdk:153.0.0 gcloud version --format='value("Google Cloud SDK")'
+153.0.0
+```
+
+### Installing additional components
+
+If you need to run additional gcloud components, you need to extend the base image:
+
+For example, for appengine-python:
+```dockerfile
+FROM google/cloud-sdk
+RUN gcloud components install app-engine-python
+```
 
-    # Get the cloud sdk image:
-    $ docker pull google/cloud-sdk
+or for app-engine-java
+```
+FROM google/cloud-sdk
+RUN apk --update add openjdk7-jre
+RUN gcloud components install app-engine-java
+WORKDIR /apps
+```
 
-    # Auth & save the credentials in gcloud-config volumes:
-    $ docker run -t -i --name gcloud-config google/cloud-sdk gcloud init
+### Volume mounts
 
-    # If you would like to use service account instead please look here:
-    $ docker run -t -i --name gcloud-config google/cloud-sdk gcloud auth activate-service-account <your-service-account-email> --key-file /tmp/your-key.p12 --project <your-project-id>
+For a volume mount using credentials:
+```
+docker run -t -i --name gcloud-config google/cloud-sdk gcloud auth login
+```
+then
+```
+docker run --rm -ti --volumes-from gcloud-config google/cloud-sdk gcloud compute instances list --project your_project
+NAME        ZONE           MACHINE_TYPE   PREEMPTIBLE  INTERNAL_IP  EXTERNAL_IP      STATUS
+instance-1  us-central1-a  n1-standard-1               10.240.0.2   8.34.219.29      RUNNING
+```
 
-    # Re-use the credentials from gcloud-config volumes & run sdk commands:
-    $ docker run --rm -ti --volumes-from gcloud-config google/cloud-sdk gcloud info
-    $ docker run --rm -ti --volumes-from gcloud-config google/cloud-sdk gcloud components list
-    $ docker run --rm -ti --volumes-from gcloud-config google/cloud-sdk gcloud compute instances list
-    $ docker run --rm -ti --volumes-from gcloud-config google/cloud-sdk gsutil ls
+### Google Appengine base
```

Comment:
I doubt if this section is necessary at all. You can move this as a note under `### Installing additional components`